Documentation/block.md

```diff
@@ -35,8 +35,13 @@ metadata:
 provisioner: ceph.rook.io/block
 parameters:
   pool: replicapool
+<<<<<<< HEAD
   #The value of "clusterNamespace" MUST be the same as the one in which your rook cluster exist
   clusterNamespace: rook-ceph
+=======
+  clusterName: rook-ceph
+  fstype: xfs
+>>>>>>> Added fstype: xfs as "default" example to bring Rook in line with Ceph recommendations
```

Comment:
@patrickstjohn It seems that you wrongly rebased, could you fix that (`clusterNamespace`) is the correct one.